test/clj/rems/testing_util.clj

```diff
@@ -54,3 +54,8 @@
                                         (organizations/get-all-organization-roles ~user)
                                         (applications/get-all-application-roles ~user))]
      ~@body))
+
+(defmacro with-fake-login-users [users & body]
+  `(with-redefs [rems.auth.fake-login/get-fake-login-users (constantly ~users)]
+     ~@body))
```

Comment:
i'm not happy to use a macro, but this seemed like a plausible solution to succinctly swap in desired login data wherever needed. we have some other similar wrappers but in the current form the composing aesthetic is a bit XML-like.

Reply:
Since they don't come from the database, then I guess the `with-redefs` is the way to go. I guess we use it mostly plain but this wrapper seems OK. Maybe it can be more generic once we put the OIDC/login refactoring in so that it can be used no matter what the login method was (and tests can do fun stuff).